flake.nix

```diff
@@ -44,7 +44,7 @@
     # tailscaleRev is the git commit at which this flake was imported,
     # or the empty string when building from a local checkout of the
     # tailscale repo.
-    tailscaleRev = if builtins.hasAttr "rev" self then self.rev else "";
+    tailscaleRev = self.rev or "";
```

Comment:
I also took the liberty of shortening this; if `self.rev` doesn't exist (i.e. because the git tree is dirty), it will still fall back to the empty string, just as before.

Reply:
Nope, looks fine to me! I didn't know that worked in Nix )